frontend/src/App.js

```diff
@@ -1,48 +1,73 @@
-import "./App.css";
-import Dropdown from "./components/Dropdown";
-import fetchAllRoutes from "./utils/fetchAllRoutes";
-import { useEffect, useState } from "react";
+import './App.css';
+import Dropdown from './components/Dropdown';
+import fetchAllRoutes from './utils/fetchAllRoutes';
+import { useEffect, useState } from 'react';
+import ClusterCard from './components/ClusterCard';
 
 function App() {
   const [allRoutes, setAllRoutes] = useState([]);
   const [selectedRoute, setSelectedRoute] = useState('');
   const [routeDetails, setRouteDetails] = useState(null);
 
   useEffect(() => {
-    const fetch = async () => {
+    const fetchRoutes = async () => {
       try {
         const fetchedRoutes = await fetchAllRoutes();
         setAllRoutes(fetchedRoutes);
       } catch (error) {
-        console.error("Failed to fetch routes:", error);
+        console.error('Failed to fetch routes:', error);
       }
     };
-    fetch();
+
+    fetchRoutes();
   }, []);
 
   useEffect(() => {
-    if (selectedRoute) {
-      const route = allRoutes.find(route => route.routeId === parseInt(selectedRoute, 10));
-      setRouteDetails(route);
-    }
+    const fetchRouteDetails = async () => {
+      if (selectedRoute) {
+        try {
+          const route = allRoutes.find((route) => route.routeId === parseInt(selectedRoute, 10));
+          setRouteDetails(route);
+        } catch (error) {
+          console.error('Failed to fetch route details:', error);
+        }
+      }
+    };
+
+    fetchRouteDetails();
   }, [selectedRoute, allRoutes]);
 
   return (
     <div className="App">
       <h1>Route Details</h1>
-      <Dropdown 
+      <Dropdown
         routes={allRoutes}
         selectedRoute={selectedRoute}
         setSelectedRoute={setSelectedRoute}
         showId={false}
       />
-      {routeDetails && (
+      {routeDetails ? (
         <div>
-          <h2>{routeDetails.name}</h2>
+          <p>Name: {routeDetails.name}</p>
           <p>ID: {routeDetails.routeId}</p>
-          <p>Sequence: {routeDetails.sequence}</p>
-          <p>Object: {routeDetails.object}</p>
+          <p>Technical Level: {routeDetails.technicalLevelName || 'N/A'}</p>
```

Comment:
We don't need this information on the page. Let's delete these and just show the cluster cards